packages/fourwings-aggregate/src/aggregate-cell.ts

```diff
@@ -6,7 +8,8 @@ const aggregateCell = (
   delta: number,
   quantizeOffset: number,
   numSublayers: number,
-  debug = false
+  aggregationOperation = AggregationOperation.sum,
+  multiplier = VALUE_MULTIPLIER
```

Comment:
This https://github.com/GlobalFishingWatch/frontend/blob/fourwings-aggregate/env-layers/packages/react-hooks/src/use-map-interaction/use-map-interaction.ts#L44 is still using the old params format order.
At least we need to remove the `debug` param to use the default ones.

Which points me that there are a lot of params, and it enforces to use the not required ones at the end. What about moving to a configuration object and destructure like:
```js
const aggregateCell = ({
  rawValues: string,
  frame: number,
  delta: number,
  quantizeOffset: number,
  numSublayers: number,
  aggregationOperation: AggregationOperation.sum,
  multiplier: VALUE_MULTIPLIER
}) => {
```
I like it because when using it the order doesn't matter anymore and it is easier to read what the function params means:
```js
const values = aggregateCell(
  rawValues: properties.rawValues,
  frame,
  delta: timeChunks.deltaInIntervalUnits,
  quantizeOffset: activeTimeChunk.quantizeOffset,
  numSublayers,
)
```